cmd/thanos/rule.go

```diff
@@ -626,7 +641,7 @@ func runRule(
 		})
 	}
 
-	confContentYaml, err := objStoreConfig.Content()
+	confContentYaml, err = objStoreConfig.Content()
 	if err != nil {
 		return err
 	}
```

Comment:
I guess we don't need this whole block as we already have the required `confContentYaml`.

Reply:
I do not think we already have `confContentYaml` though. This is the block that gets it

Reply:
Check line 576